packages/@webex/webex-core/src/lib/credentials/credentials.js

```diff
@@ -528,9 +548,7 @@ const Credentials = WebexPlugin.extend({
         this.scheduleRefresh(this.supertoken.expires);
       })
       .catch((error) => {
-        const {InvalidRequestError} = grantErrors;
-
-        if (error instanceof InvalidRequestError) {
+        if (error instanceof OAuthError) {
           // Error: The refresh token provided is expired, revoked, malformed, or invalid. Hence emit an event to the client, an opportunity to logout.
```

Comment:
If you are increasing the scope of when we will be logged out, this comment is no longer fully accurate